src/main/resources/public/ts/sniplets/structure-option-is-average-skills.ts

```diff
@@ -0,0 +1,50 @@
+
+import {
+    structureOptionsService,
+    IStructureOptionsService,
+    StructureOptionsService,
+    StructureOptions
+} from "../services";
+
+import {toasts} from "entcore";
+import {safeApply} from "../utils/functions/safeApply";
+
+export class SnipletStructureOptions {
+   option : StructureOptions;
+
+    constructor( private $scope: any, private structureOptionsService: IStructureOptionsService) {
+        this.initOptionsIsAverageSkills($scope.source.idStructure);
+    }
+
+    async initOptionsIsAverageSkills(structureId: String): Promise<void> {
```

Comment:
Il faudrait éviter l'usage de l'objet String et plutôt utiliser primitive `string`

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html